serverlessworkflow/modules/ROOT/pages/cloud/operator/install-kn-workflow-cli.adoc

```diff
@@ -0,0 +1,90 @@
+= Installing the Knative Workflow Plugin
+:compat-mode!:
+// Metadata:
+:description: Install the operator on Kubernetes clusters
+:keywords: kogito, sonataflow, workflow, serverless, operator, kubernetes, minikube, openshift, containers
+// links
+
+*Prerequisites*
+
+https://75129--ocpdocs-pr.netlify.app/openshift-serverless/latest/install/installing-kn
+
+* You have first installed the  link:https://76490--ocpdocs-pr.netlify.app/openshift-serverless/latest/install/installing-kn[Knative CLI].
```

Comment:
This should be a link to official community documentation of knative

Reply:
yes, we can change that link.
But please note that this PR is just a draft to share what the proposal of installing the kn workflow plugin was. IDK if we are publishing that image in community·